src/Libraries/RevitNodes/GeometryConversion/ProtoToRevitFace.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Autodesk.DesignScript.Runtime;
+using Edge = Autodesk.DesignScript.Geometry.Edge;
+using Face = Autodesk.DesignScript.Geometry.Face;
+using ProtoPt = Autodesk.DesignScript.Geometry.Point;
+using Solid = Autodesk.DesignScript.Geometry.Solid;
+using Surface = Autodesk.DesignScript.Geometry.Surface;
+
+namespace Revit.GeometryConversion
+{
+   [SupressImportIntoVM]
+   public static class ProtoToRevitSolid
```

Comment:
Yes, I agree that `DynamoToRevitBRep` is better. We already have a couple of classes/files named `ProtoToRevit*`. Is it OK to rename those too while we at it?